### DISCUSSION TITLE
Header files are not shown in the project outline after a small edit !

### DISCUSSION THREAD

**Discussion by omarhatem97** [microsoft/vscode-cmake-tools#3800] [Q&A] [answered]
So when you are working in a cmake project. it's sometimes a good idea if you navigate and explore your project using the project outline found in cmake tools tab.
in general if you add your headers to an executable or a lib as in the line below:
`add_executable(${PROJECT_NAME}_exe  ${SOURCES} ${HEADERS})`
it will view the header files alongside the source files in the project files, but for a weired reason, if you add grpc sources, when working with grpc, all the header files are not shown!
`add_executable(${PROJECT_NAME}_exe  ${SOURCES} ${HEADERS} ${GRPC_SRCS})`
does anybody have a clue ??

**Comment by gcampbell-msft** [comment, marked as answer]
This sounds like it *may* be a bug, so it would be incredibly helpful if you could create an issue for this with repro steps and a full repro project so that we can investigate it. 

Once this issue is created we'd be more than happy to investigate!
